<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="prakash-PC" name="Default test" tests="2" failures="0" timestamp="21 Jan 2015 16:52:08 GMT" time="10.463" errors="0">
  <testcase name="@BeforeClass configBeforeClass" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <failure type="java.io.FileNotFoundException" message="E:\prakash_Selenium\Test_Data.xls (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: E:\prakash_Selenium\Test_Data.xls (The system cannot find the file specified)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at java.io.FileInputStream.<init>(FileInputStream.java:101)
	at com.actitime.genericLib.ExcelLib.getExcelData(ExcelLib.java:20)
	at com.actitime.projectandcustomerTest.ProjectAndCustomerTest.configBeforeClass(ProjectAndCustomerTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- @BeforeClass configBeforeClass -->
  <testcase name="@BeforeMethod configBeforeMtd" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod configBeforeMtd -->
  <testcase name="@AfterMethod configAfterMtd" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- @AfterMethod configAfterMtd -->
  <testcase name="@BeforeMethod configBeforeMtd" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod configBeforeMtd -->
  <testcase name="@AfterMethod configAfterMtd" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- @AfterMethod configAfterMtd -->
  <testcase name="@AfterClass configAfterClass" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- @AfterClass configAfterClass -->
  <testcase name="createCustomer" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- createCustomer -->
  <testcase name="modifyCustomer" time="0.0" classname="com.actitime.projectandcustomerTest.ProjectAndCustomerTest">
    <skipped/>
  </testcase> <!-- modifyCustomer -->
</testsuite> <!-- Default test -->
